pkg/controller/qos.go

```diff
@@ -47,6 +51,20 @@ func QosPolicyLogger(log *logrus.Logger, qos *qospolicy.QosPolicy) *logrus.Entry
 	})
 }
 
+func qosInit(cont *AciController, stopCh <-chan struct{}) {
+	cont.log.Debug("Initializing qos client")
+	restconfig := cont.env.RESTConfig()
+	qosClient, err := qosclientset.NewForConfig(restconfig)
+	if err != nil {
+		cont.log.Errorf("Failed to intialize qos client")
+		return
+	}
+	cont.initQosInformerFromClient(qosClient)
+	cache.WaitForCacheSync(stopCh, cont.qosInformer.HasSynced)
```

Comment:
can we move the informer run before the waitForCache Sync

Reply:
I think I will eliminate the WaitForCacheSync. It doesn't serve any purpose in this workflow actually.